lib/bucky/test_equipment/pageobject/base_pageobject.rb

```diff
@@ -54,7 +54,22 @@ def find_elem(method, value)
 
           elements = @driver.find_elements(method_name, value)
           raise_if_elements_empty(elements, method_name, value)
-          elements.first.instance_eval { define_singleton_method('[]') { |num| elements[num] } }
+
+          get_element_or_attribute = lambda do |elems, arg|
```

Comment:
How about making this Lambda into the private method?

Maybe it will be more easier to read.

Reply:
Thank you for review.
I tried it, but can't call private method in `instance_eval`.

Reply:
OK I understand!